src/Sylius/Behat/Context/Ui/PaypalContext.php

```diff
@@ -73,4 +112,33 @@ public function iCancelMyPaypalPayment()
     {
         $this->paypalExpressCheckoutPage->cancel();
     }
+
+    /**
+     * @When I try to pay again
+     */
+    public function iTryToPayAgain()
+    {
+        $order = $this->getLastOrder();
+        $payment = $order->getLastPayment();
+        $this->paypalApiMocker->mockApiPaymentInitializeResponse();
+        $this->orderPaymentsPage->clickPayButtonForGivenPayment($payment);
+    }
+
+    /**
+     * @return OrderInterface
+     *
+     * @throws \RuntimeException
+     */
+    private function getLastOrder()
+    {
+        $customer = $this->sharedStorage->get('user')->getCustomer();
+        $orders = $this->orderRepository->findByCustomer($customer);
```

Comment:
No problem in Behats because there is only this one or 2 orders in db, but we should remember about this.